jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/resource/webappexception/mapper/JAXRSClientIT.java

```diff
@@ -30,9 +30,9 @@
 import org.junit.jupiter.api.TestInfo;
 import org.junit.jupiter.api.extension.ExtendWith;
 
-import jakarta.ws.rs.core.Response.Status;
 import ee.jakarta.tck.ws.rs.common.JAXRSCommonClient;
 import ee.jakarta.tck.ws.rs.lib.util.TestUtil;
+import jakarta.ws.rs.core.Response.Status;
```

Comment:
Change 
Copyright (c) 2012, 2020, 2021
to
Copyright (c) 2012, 2024

Reply:
No, I was incorrect.   The practices for Eclipse projects are different.   https://www.eclipse.org/projects/handbook/#ip-copyright-headers

So 
Copyright (c) 2012, 2020, 2021
should really be changed to
Copyright (c) 2012

Reply:
Strange, the document used to allow for a single year, or two years (creation, last modification), and in that manner the maven copyright plugin checks two year-values in the copyright header. I am not a lawyer, but I saw somewhere that a copyright wears off after some ten years, unless updated with a new code (and year).

Reply:
From the same document, FAQ section:
> Do we need to specify a range of years in the copyright statement?

> No. In the past, legal advice was that the year of the initial creation of the content and the year of the last change should be reflected in the copyright header. This is no longer the case. Specify the year that the content was initially created in the copyright statement.